admin/backend/main.py

```
@@ -0,0 +1,95 @@
+"""
+Main FastAPI application for the RAG admin dashboard backend.
+"""
+
+import os
+
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+from fastapi.responses import FileResponse
+from fastapi.staticfiles import StaticFiles
+
+from .database import db_manager
```

Ruff (0.12.2)

`.database.db_manager` imported but unused

Remove unused import: `.database.db_manager`

(F401)